tests/examples/java_examples_test.go

```diff
@@ -90,6 +90,13 @@ func TestExamples(t *testing.T) {
 					assert.True(t, strings.Contains(publicHostName, "."))
 				},
 			})
+
+		test.StackName = "awsjavawebserverstack"
```

Comment:
StackName must be set because a randomly chosen stack name currently confuses the record-replay framework and it fails to replay. Unfortunate.